2023-04-22 15:04:32,369 INFO    MainThread:2190002 [wandb_setup.py:_flush():76] Configure stats pid to 2190002
2023-04-22 15:04:32,369 INFO    MainThread:2190002 [wandb_setup.py:_flush():76] Loading settings from /home/s1797743/.config/wandb/settings
2023-04-22 15:04:32,369 INFO    MainThread:2190002 [wandb_setup.py:_flush():76] Loading settings from /home/s1797743/thesis/final/AAAsegmentor/notebooks/wandb/settings
2023-04-22 15:04:32,369 INFO    MainThread:2190002 [wandb_setup.py:_flush():76] Loading settings from environment variables: {'_require_service': 'True'}
2023-04-22 15:04:32,370 INFO    MainThread:2190002 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2023-04-22 15:04:32,370 INFO    MainThread:2190002 [wandb_init.py:_log_setup():461] Logging user logs to /home/s1797743/thesis/final/AAAsegmentor/notebooks/wandb/run-20230422_150432-39tygsr5/logs/debug.log
2023-04-22 15:04:32,370 INFO    MainThread:2190002 [wandb_init.py:_log_setup():462] Logging internal logs to /home/s1797743/thesis/final/AAAsegmentor/notebooks/wandb/run-20230422_150432-39tygsr5/logs/debug-internal.log
2023-04-22 15:04:32,370 INFO    MainThread:2190002 [wandb_init.py:init():495] calling init triggers
2023-04-22 15:04:32,370 INFO    MainThread:2190002 [wandb_init.py:init():498] wandb.init called with sweep_config: {}
config: {}
2023-04-22 15:04:32,370 INFO    MainThread:2190002 [wandb_init.py:init():514] re-initializing run, found existing run on stack: 3gf4k5c8
2023-04-22 15:04:32,372 INFO    MainThread:2190002 [wandb_run.py:_finish():1728] finishing run barisimre/debug_cluster/3gf4k5c8
2023-04-22 15:04:32,372 INFO    MainThread:2190002 [jupyter.py:save_history():435] not saving jupyter history
2023-04-22 15:04:32,373 INFO    MainThread:2190002 [jupyter.py:save_ipynb():377] not saving jupyter notebook
2023-04-22 15:04:32,373 INFO    MainThread:2190002 [wandb_init.py:_jupyter_teardown():393] cleaning up jupyter logic
2023-04-22 15:04:32,373 INFO    MainThread:2190002 [wandb_run.py:_atexit_cleanup():1931] got exitcode: 0
2023-04-22 15:04:32,373 INFO    MainThread:2190002 [wandb_run.py:_restore():1914] restore
2023-04-22 15:04:32,373 INFO    MainThread:2190002 [wandb_run.py:_restore():1920] restore done
2023-04-22 15:04:32,382 INFO    MainThread:2190002 [wandb_run.py:_on_finish():2221] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 715
  total_bytes: 715
}

2023-04-22 15:04:32,695 INFO    MainThread:2190002 [wandb_run.py:_on_finish():2221] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 715
  total_bytes: 715
}

2023-04-22 15:04:33,178 INFO    MainThread:2190002 [wandb_run.py:_on_finish():2221] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 715
  total_bytes: 715
}

2023-04-22 15:04:33,802 INFO    MainThread:2190002 [wandb_run.py:_on_finish():2221] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 715
  total_bytes: 4158
}

2023-04-22 15:04:33,908 INFO    MainThread:2190002 [wandb_run.py:_on_finish():2221] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 715
  total_bytes: 4158
}

2023-04-22 15:04:34,012 INFO    MainThread:2190002 [wandb_run.py:_on_finish():2221] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3584
  total_bytes: 4158
}

2023-04-22 15:04:34,119 INFO    MainThread:2190002 [wandb_run.py:_on_finish():2221] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4158
  total_bytes: 4158
}

2023-04-22 15:04:34,225 INFO    MainThread:2190002 [wandb_run.py:_on_finish():2221] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4158
  total_bytes: 4158
}

2023-04-22 15:04:34,329 INFO    MainThread:2190002 [wandb_run.py:_on_finish():2221] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4158
  total_bytes: 4158
}

2023-04-22 15:04:34,433 INFO    MainThread:2190002 [wandb_run.py:_on_finish():2221] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4158
  total_bytes: 4158
}

2023-04-22 15:04:35,061 INFO    MainThread:2190002 [wandb_run.py:_on_finish():2221] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4158
  total_bytes: 4158
}

2023-04-22 15:04:35,457 INFO    MainThread:2190002 [wandb_run.py:_on_finish():2221] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4158
  total_bytes: 4158
}
local_info {
}

2023-04-22 15:04:36,570 INFO    MainThread:2190002 [wandb_run.py:_footer_history_summary_info():3305] rendering history
2023-04-22 15:04:36,570 INFO    MainThread:2190002 [wandb_run.py:_footer_history_summary_info():3337] rendering summary
2023-04-22 15:04:36,579 INFO    MainThread:2190002 [wandb_run.py:_footer_sync_info():3261] logging synced files
2023-04-22 15:04:36,588 INFO    MainThread:2190002 [wandb_init.py:init():548] starting backend
2023-04-22 15:04:36,589 INFO    MainThread:2190002 [wandb_init.py:init():552] setting up manager
2023-04-22 15:04:36,595 INFO    MainThread:2190002 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2023-04-22 15:04:36,600 INFO    MainThread:2190002 [wandb_init.py:init():558] backend started and connected
2023-04-22 15:04:36,618 INFO    MainThread:2190002 [wandb_run.py:_label_probe_notebook():1105] probe notebook
2023-04-22 15:04:36,618 INFO    MainThread:2190002 [wandb_run.py:_label_probe_notebook():1115] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2023-04-22 15:04:36,618 INFO    MainThread:2190002 [wandb_init.py:init():636] updated telemetry
2023-04-22 15:04:36,632 INFO    MainThread:2190002 [wandb_init.py:init():667] communicating run to backend with 30 second timeout
2023-04-22 15:04:37,981 INFO    MainThread:2190002 [wandb_run.py:_on_init():1982] communicating current version
2023-04-22 15:04:38,035 INFO    MainThread:2190002 [wandb_run.py:_on_init():1986] got version response upgrade_message: "wandb version 0.15.0 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2023-04-22 15:04:38,035 INFO    MainThread:2190002 [wandb_init.py:init():700] starting run threads in backend
2023-04-22 15:04:38,098 INFO    MainThread:2190002 [wandb_run.py:_console_start():1962] atexit reg
2023-04-22 15:04:38,098 INFO    MainThread:2190002 [wandb_run.py:_redirect():1820] redirect: SettingsConsole.WRAP_RAW
2023-04-22 15:04:38,098 INFO    MainThread:2190002 [wandb_run.py:_redirect():1885] Wrapping output streams.
2023-04-22 15:04:38,098 INFO    MainThread:2190002 [wandb_run.py:_redirect():1907] Redirects installed.
2023-04-22 15:04:38,098 INFO    MainThread:2190002 [wandb_init.py:init():732] run started, returning control to user process
